src/main/io/osd.c

```diff
@@ -143,7 +148,27 @@ typedef struct statistic_s {
     uint8_t min_link_quality;
 } statistic_t;
 
+typedef struct radioControls_s {
+    rc_alias_e left_vertical;
```

Comment:
(`uint8_t` may be used to save some space, enum in 32bit)

Reply:
Done.